src/Components/TabItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../types/Tab';
+
+interface Props {
+  tab: Tab;
+}
+
+export const TabItem: FC<Props> = ({ tab }) => {
+  const { tabId } = useParams();
+
+  return (
+    <li
+      key={tab.id}
+      data-cy="Tab"
+      className={cn({ 'is-active': tabId === tab.id })}
+    >
+      <Link to={`/tabs/${tab.id}`}>
```

Comment:
```suggestion
      <Link to={tab.id}>
```

Reply:
@ruslanpashkov thanks, I missed that moment

Reply:
@ruslanpashkov it's a good idea, but now I don't understand why it doesn't work, I need to check it. Now if I just put `tab.id` like you say, I have this moment `localhost:3000/#/tabs/tab-2/tab-3`

Reply:
@vladyslav-lapshyn you can try to create only one `<Route path="/">` and put other routers inside it
```
<Routes>
  <Route path="/">
    <Route index element={<HomePage />} />
    <Route path="home" element={<Navigate to="/" replace />} />
    <Route path="tabs" element={<TabsPage />}>
      <Route path=":tabId" element={<TabsPage />} />
    </Route>
    <Route path="*" element={<NotFoundPage />} />
  </Route>
</Routes>
```

Reply:
@ruslanpashkov wow, you're right. Thanks 🔥